### DISCUSSION TITLE
Why there are no commits on the repository for more than 8 months?

### DISCUSSION THREAD

**Discussion by kleninmaxim** [dogecoin/dogecoin#3238] [Help needed] [answered]
Hi, everyone!

Why there are no commits on the repository for more than 8 months?
Why are there no updates?
And why are issues and pull requests not fixed?

**Comment by chromatic** [comment, marked as answer]
We're doing development primarily on the [1.14.7-dev](https://github.com/dogecoin/dogecoin/discussions/3238) branch, to prepare for the 1.14.7 release.

You can look at [closed issues](https://github.com/dogecoin/dogecoin/issues?q=is%3Aissue+is%3Aclosed) and [merged pull requests](https://github.com/dogecoin/dogecoin/pulls?q=is%3Apr+is%3Aclosed) to see recent activity.

Of course, some things happen behind the scenes. Patrick has been working on the security disclosure for quite some time to make sure that Dogecoin isn't vulnerable, that the disclosure doesn't put the network or operators at risk, and that other coins have time to evaluate their own security.

Is there a specific issue or pull request you're most interested in? I'm happy to take a look and perhaps solicit other developers or interested people to help out.

**Reply by kleninmaxim** [reply]
Thanks for the answer!
I have no questions any more
